posts/how-to-integrate-aws-connect.md

```diff
@@ -0,0 +1,177 @@
+        loginPopupAutoClose: true,
+        softphone: {
+            allowFramedSoftphone: true
+        },
+        pageOptions: {
+            enableAudioDeviceSettings: true,
+            enablePhoneTypeSettings: true
+        }
+    })
+    ```
+
+5. Hook in to [Connect events](https://github.com/amazon-connect/amazon-connect-streams/blob/master/Documentation.md).
+
+    ```js
+    // When an agent is logs in
+    connect.agent(agent => {
+        // gather information about the agent
+        const agentName = agent.getName()
+        const agentContacts = agent.getContacts()
+        const agentPermissions = agent.getPermissions()
+    })
+
+    // On inbound communication
+    connect.contact(contact => {
+        // receive contact metadata
+        const contactAttributes = contact.getAttributes()
+    })
+    ```
+
+## Configuring your AWS Connect Instance
+
+The last step is to specify an **approved origin**. This is the base URL, and hence application(s), that you are giving permission to integrate with your newly created Connect instance. Follow the below steps to specify one or more approved origins based on your needs:
+
+1. Go to the Account overview page for your instance by clicking the `Instance alias` name from the Instances table
+2. Select `Approved origins` from the left-side navigation menu
+3. Add the base URL for your web application. Examples include:
+   - `https://production-url`
+   - `https://development-url`
+   - `http://localhost:4200`
+
+Log in to your newly created Connect instance using the credentials that you set up above in step three. The first time that you log in you will be presented a Dashboard page with a step-by-step tutorial on how to configure and tune your new Cloud contact center.
+
+You should now be able to run your application after setting the approved origins.  The image below shows a runnable demo (source available [here](https://gitlab.ippon.fr/jstrickler/aws-connect-demo)).
+
+![Local Demo](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/05/connect-local-demo.png)
+
+## Experience your new functionality
+
+Now that you're set up, let's do a brief run through of the functionality that you've just unlocked.
+
+### Receive a phone call
+
+In order to receive a phone call, you first need to **claim a phone number**:
+
+- Log in to your Connect admin console at *https://YOUR_INSTANCE_ID.my.connect.aws*
+  - Use the admin credentials from your Connect setup (not your AWS account creds)
+- On the left hand menu, navigate to Routing -> Phone Numbers
+- Click "Claim a number" from the top-left buttons
+- Create either a toll-free or a DID (direct inward dialing) phone number
+
+Now you have a phone number.  You're ready to receive inbound calls.  Go to your application that you've intregated with the CCP and set yourself to **available** from the top-most dropdown menu.  Now call freshly-claimed phone number and it will appear as an inbound call!  
```

Comment:
```suggestion
Now you have a phone number, you are ready to receive inbound calls.  Go to your application that you have integrated with the CCP and set yourself to **available** from the top-most dropdown menu.  Now call freshly-claimed phone number and it will appear as an inbound call!  
```